### PR TITLE
Feature/db

### PR DESCRIPTION
**Pull request by CrashingBrain**
Implemented tests for the DB models. 
Modified the Seed to be randomly generated. The only thing is, some fields like "title" and "category Name" don't make much sense because they are random words (kind of on topic tho)
Also "address" is the name of a random city in the world

### CODE REVIEW

File: seed_data/seedData.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var mongoose = require('mongoose');
 var JobListing = mongoose.model('Freelance');
 var Review = mongoose.model('Review');
 var ObjectId = mongoose.Types.ObjectId;
-
+/* OLD SEED
```
Comment (BalestraPatrick) [inline seed_data/seedData.js]: Please delete the old code. We can always go back and find it in git.

File: seed_data/categoriesData.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+var categoriesData = {
+  name: 'Category',
+  data: [
+    {"_id":"58cc4b15fc13ae5ec7000122","categoryName":"Support","freelancers":[]},
```
Comment (zuccha) [inline seed_data/categoriesData.js:4]: Add spaces after commas for readability.

File: seed_data/tagsData.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+var tagsData = {
+  name : 'Tag',
+  data : [
+    {"_id":"58cc4baefc13ae66e40000cd","tagName":"implementation","freelancers":[]},
+    {"_id":"58cc4baefc13ae66e40000ce","tagName":"definition","freelancers":[]},
+    {"_id":"58cc4baefc13ae66e40000cf","tagName":"Grass-roots","freelancers":[]},
+    {"_id":"58cc4baefc13ae66e40000d0","tagName":"full-range","freelancers":[]},
```
Comment (zuccha) [inline seed_data/tagsData.js:7]: Add commas for readability. Also, make all tag names lowercase for consistency.

File: seed_data/categoriesData.js (hunk 2 above)
Comment (aronfiechter) [reply seed_data/categoriesData.js:4]: Even if it's a seed?

Comment (CrashingBrain) [reply seed_data/categoriesData.js:4]: it's not really meant to be readable in that form but I can add it anyway...

File: test/models/constructAndCreate/Review.js [hunk 4]
```diff
@@ -1 +1,141 @@
...
+	  review.title = 'Sweet product';
+	  review.text = 'GG, well played';
+	  review.date = Date.now();
+      utils.errorIfNullUndefinedOrEmpty(review, 'score', done );
+    });
+
+      it ('should bound over 5 avgScore to 5', function(done){
+		  let review = new Review();
+		  review.author = 'Bob Knopfler';
+		  review.title = 'Sweet product';
```
Comment (zuccha) [inline test/models/constructAndCreate/Review.js]: There is something wrong with indentation here.

File: test/models/constructAndCreate/Review.js [hunk 5]
```diff
@@ -1 +1,141 @@
...
+        should.not.exist(err, 'No error should occur');
+        review.score.should.equal(5);
+        done();
+      });
+    });
+
+      it ('should bound less than 0 avgScore to 0', function(done){
+		  let review = new Review();
+		  review.author = 'Bob Knopfler';
+		  review.title = 'Sweet product';
```
Comment (zuccha) [inline test/models/constructAndCreate/Review.js]: Wrong indentation here too.

**Review by zuccha — COMMENTED** [review]
Some suggestions about formatting.

File: test/models/constructAndCreate/Review.js (hunk 4 above)
Comment (CrashingBrain) [reply test/models/constructAndCreate/Review.js]: corrected (copy-paste issue)

File: test/models/constructAndCreate/Review.js (hunk 5 above)
Comment (CrashingBrain) [reply test/models/constructAndCreate/Review.js]: corrected (copy-paste issue)

**Pull request merged**